### DISCUSSION TITLE
How to turn off the dashlines indicating indent areas in insert mode?

### DISCUSSION THREAD

**Discussion by hopezh** [lukas-reineke/indent-blankline.nvim#630] [Q&A] [answered]
As highlighted by the arrow in the image below, three dash lines `---` show up in the indent area entering the `insert` mode. (Not related to the `space_char_blanklin` option which sets the char for indent spaces in `normal` mode.) 

May I ask if this is related to `indent-blankline`? And, if it is, which option I shall use to disable it? 

Thanks. 

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/23435810/8e782626-48d8-46e0-b01f-0b09c8051c3f)

**Comment by lukas-reineke** [comment, marked as answer]
This is not indent-blankline, looks like `:help lcs-trail` maybe?
